metaflow_extensions/card_notebook/plugins/cards/notebook/__init__.py

```diff
@@ -44,13 +44,14 @@ def render(self, task):
 
          # inject `run_id`, `task_id` and `flow_name`` into the parameters
         run_id = task.parent.parent.id
+        step_name = task.path_components[-2]
```

Comment:
@valayDave this is the line of code that I'm unsure about the most, is this the most robust way of getting the `step_name` ?